### ISSUE TITLE
Taskify : Make Achievements Section Responsive

### ISSUE THREAD

**Issue by Rajarshi12321** [DSCKGEC/Taskify#9]
- Make achievements section responsive in the landing page

**Comment by Rik-21** [comment]
can i work on this issue

**Comment by Rupam-Matabber** [comment]
@Rik-21 Yes you can work on the issue but before that, you need to be assigned. Do you want me to assign you to the issue?
